packages/react-reconciler/src/ReactUpdateQueue.js

```diff
@@ -219,21 +215,9 @@ function appendUpdateToQueue<State>(
     queue.lastUpdate.next = update;
     queue.lastUpdate = update;
   }
-  if (
-    queue.expirationTime === NoWork ||
-    queue.expirationTime > expirationTime
-  ) {
-    // The incoming update has the earliest expiration of any update in the
-    // queue. Update the queue's expiration time.
-    queue.expirationTime = expirationTime;
-  }
 }
 
-export function enqueueUpdate<State>(
-  fiber: Fiber,
-  update: Update<State>,
-  expirationTime: ExpirationTime,
-) {
+export function enqueueUpdate<State>(fiber: Fiber, update: Update<State>) {
```

Comment:
Hey @acdlite, I was using `expirationTime` for interaction tracking (#13234)

Any objection to my adding the parameter back? Or any alternative approach you'd recommend instead?

Reply:
Disregard! Will move the interaction tracking check to `scheduleWork` instead.